index.html

```diff
@@ -31,9 +33,42 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>
+            <div id="header">
```

Comment:
`header`, `content`, and `footer` are too broad for `id` names with so many others in the class. Favor a single `id` for your `div`, rather than individual `id`s for each piece of that `div`!